networkx/tests/test_all_random_functions.py

```diff
@@ -189,8 +189,8 @@ def kernel_integral(u, w, z):
     t(nx.extended_barabasi_albert_graph, n, m, p, q, seed=seed)
     t(nx.powerlaw_cluster_graph, n, m, p, seed=seed)
     t(nx.random_lobster_graph, n, p1, p2, seed=seed)
-    t(nx.random_powerlaw_tree, n, seed=seed, tries=5000)
-    t(nx.random_powerlaw_tree_sequence, 10, seed=seed, tries=5000)
+    t(nx.random_powerlaw_tree, 5, seed=seed, tries=5000)
+    t(nx.random_powerlaw_tree_sequence, 5, seed=seed, tries=5000)
```

Comment:
Any particular reason for changing the `n` here?

Reply:
These tests exceed the maximum number of tries with the old values now that invalid sequences are filtered more thoroughly.